packages/web-app-files/src/mixins/actions/navigate.ts

```diff
@@ -52,6 +52,15 @@ export default {
           canBeDefault: true,
           componentType: 'router-link',
           route: ({ resources }) => {
+            if (
+              isShareSpaceResource(this.space) &&
```

Comment:
I think you need to use `this.$_navigate_getSpace` here instead of `this.space`.

Reply:
`this.space` is actually always set in this scenario, at least on those 2 relevant views.

Reply:
But how? The `shared with others` and `shared via link` page don't have one common parent space.

Reply:
But it's not available as `this.space`.  The linked function is used for binding the space into the context menu: https://github.com/owncloud/web/blob/d44c2bfa926b2195fb37540aecb071dd3d7ee7cd/packages/web-app-files/src/views/shares/SharedViaLink.vue#L38

The `$_navigate_getSpace` function resembles the same functionality.

Reply:
The space is given as property to `ContextActions`, which has all the actions as mixin. Isn't `space` available for the mixin then via `this`? Otherwise I don't now where that comes from 🤔

Reply:
argh. yes, you're right. 🙈